dll/win32/wininet/rsrc.rc

```diff
@@ -42,88 +42,88 @@
 #pragma code_page(65001)
 
 #ifdef LANGUAGE_BG_BG
-    #include "lang/wininet_Bg.rc"
+    #include "lang/bg_BG.rc"
 #endif
 #ifdef LANGUAGE_CS_CZ
-    #include "lang/wininet_Cs.rc"
+    #include "lang/cs_CZ.rc"
 #endif
 #ifdef LANGUAGE_DA_DK
-    #include "lang/wininet_Da.rc"
+    #include "lang/da_DK.rc"
 #endif
 #ifdef LANGUAGE_DE_DE
-    #include "lang/wininet_De.rc"
+    #include "lang/de_DE.rc"
 #endif
 #ifdef LANGUAGE_EN_US
-    #include "lang/wininet_En.rc"
+    #include "lang/en_US.rc"
 #endif
 #ifdef LANGUAGE_ES_ES
-    #include "lang/wininet_Es.rc"
+    #include "lang/es_ES.rc"
 #endif
 #ifdef LANGUAGE_EO_AA
-    #include "lang/wininet_Eo.rc"
+    #include "lang/eo_AA.rc"
 #endif
 #ifdef LANGUAGE_FI_FI
-    #include "lang/wininet_Fi.rc"
+    #include "lang/fi_FI.rc"
 #endif
 #ifdef LANGUAGE_FR_FR
-    #include "lang/wininet_Fr.rc"
+    #include "lang/fr_FR.rc"
 #endif
 #ifdef LANGUAGE_HE_IL
-    #include "lang/wininet_He.rc"
+    #include "lang/he_IL.rc"
 #endif
 #ifdef LANGUAGE_HU_HU
-    #include "lang/wininet_Hu.rc"
+    #include "lang/hu_HU.rc"
 #endif
 #ifdef LANGUAGE_IT_IT
-    #include "lang/wininet_It.rc"
+    #include "lang/it_IT.rc"
 #endif
 #ifdef LANGUAGE_JA_JP
-    #include "lang/wininet_Ja.rc"
+    #include "lang/ja_JP.rc"
 #endif
 #ifdef LANGUAGE_KO_KR
-    #include "lang/wininet_Ko.rc"
+    #include "lang/ko_KR.rc"
 #endif
 #ifdef LANGUAGE_LT_LT
-    #include "lang/wininet_Lt.rc"
+    #include "lang/lt_LT.rc"
 #endif
 #ifdef LANGUAGE_NL_NL
-    #include "lang/wininet_Nl.rc"
+    #include "lang/nl_NL.rc"
 #endif
 #ifdef LANGUAGE_NO_NO
-    #include "lang/wininet_No.rc"
+    #include "lang/no_NO.rc"
 #endif
 #ifdef LANGUAGE_PL_PL
-    #include "lang/wininet_Pl.rc"
+    #include "lang/pl_PL.rc"
 #endif
 #ifdef LANGUAGE_PT_PT
-    #include "lang/wininet_Pt.rc"
+    #include "lang/pt_PT.rc"
 #endif
 #ifdef LANGUAGE_RO_RO
-    #include "lang/wininet_Ro.rc"
+    #include "lang/ro_RO.rc"
 #endif
 #ifdef LANGUAGE_RU_RU
-    #include "lang/wininet_Ru.rc"
+    #include "lang/ru_RU.rc"
 #endif
 #ifdef LANGUAGE_SL_SI
-    #include "lang/wininet_Si.rc"
+    #include "lang/sl_SI.rc"
 #endif
 #ifdef LANGUAGE_SQ_AL
-    #include "lang/wininet_Sq.rc"
+    #include "lang/sq_AL.rc"
 #endif
 #ifdef LANGUAGE_SR_SP
-    #include "lang/wininet_Sr.rc"
+    #include "lang/sr_SP.rc"
 #endif
 #ifdef LANGUAGE_SV_SE
-    #include "lang/wininet_Sv.rc"
+    #include "lang/sv_SE.rc"
 #endif
 #ifdef LANGUAGE_TR_TR
-    #include "lang/wininet_Tr.rc"
+    #include "lang/tr_TR.rc"
 #endif
 #ifdef LANGUAGE_UK_UA
-    #include "lang/wininet_Uk.rc"
+    #include "lang/uk_UA.rc"
 #endif
 #ifdef LANGUAGE_ZH_CN
-    #include "lang/wininet_Zh.rc"
+    #include "lang/zh_CN.rc"
```

Comment:
This module is in WINESYNC, so these changes are not correct. Please revert and rename files back.